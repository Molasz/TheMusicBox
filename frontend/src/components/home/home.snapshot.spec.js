import renderer from 'react-test-renderer';
import React from 'react';
import Home from './home';
import { BrowserRouter } from 'react-router-dom';

describe('Home snapshot', () => {
  const tree = renderer.create(<Home />);
  it('Should match', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
