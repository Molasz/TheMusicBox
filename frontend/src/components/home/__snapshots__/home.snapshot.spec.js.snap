// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot Should match 1`] = `
<div
  className="home"
>
  <img
    alt="home background"
    className="home__img"
    src="https://live.staticflickr.com/8523/8675150036_8fd47b4425_b.jpg"
  />
  <h1
    className="home__title"
  >
    Welcome
  </h1>
</div>
`;
